students/wonbin/users/views.py

```diff
@@ -19,10 +19,12 @@ def post(self, request):
             password_validate(password)
             phone_number_validate(phone_number)
 
+            hashed_password =  bcrypt.hashpw(password.encode("utf-8"), bcrypt.gensalt())
```

Comment:
미리 decode하고 활용하면 더 좋을 것 같네요.
```suggestion
            hashed_password =  bcrypt.hashpw(password.encode("utf-8"), bcrypt.gensalt()).decode("utf-8")
```